README.md

```diff
@@ -25,6 +25,8 @@ Or run com.epam.aidial.core.AIDial class from your favorite IDE.
 
 ## Configuration
 
+<!--Mention that there is a Dynamic configuration an well and give a link -->
```

Comment:
Please add a link to the corresponding section and explain what kind of config to use and for which purpose

Reply:
fixed